server/text_generation_server/utils/weights.py

```diff
@@ -110,21 +111,92 @@ def get_weights_col_packed(
         prefix: str,
         block_sizes: Union[int, List[int]],
     ):
-
-        return self.weight_class(
-            weights.get_packed_sharded(
-                f"{prefix}.weight", dim=0, block_sizes=block_sizes
-            ),
+        w = weights.get_packed_sharded(
```

Comment:
My logic here is that even though some of the weights are in fp8, this should still be manageable by the default loader.
I want to avoid adding `--quantize fp8` for this type of checkpoints but this is a bit of a hack.